linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/HelpPageHandler.scala

```diff
@@ -0,0 +1,66 @@
+package io.buoyant.linkerd.admin
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{Request, Response, Status}
+import com.twitter.util.Future
+
+private[admin] class HelpPageHandler extends Service[Request, Response] {
+
+  override def apply(req: Request): Future[Response] = AdminHandler.mkResponse(html)
+
+  def html = {
```

Comment:
tiny non-blocking nit:

could use val instead of def
could omit braces for single expression